app/src/main/java/org/schabi/newpipe/util/KeyboardUtil.java

```diff
@@ -20,36 +21,20 @@ public static void showKeyboard(final Activity activity, final EditText editText
         if (activity == null || editText == null) {
             return;
         }
-
-        if (editText.requestFocus()) {
-            final InputMethodManager imm = ContextCompat.getSystemService(activity,
-                    InputMethodManager.class);
-            if (!imm.showSoftInput(editText, InputMethodManager.SHOW_FORCED)) {
-                /*
-                 * Sometimes the keyboard can't be shown because Android's ImeFocusController is in
-                 * a incorrect state e.g. when animations are disabled or the unfocus event of the
-                 * previous view arrives in the wrong moment (see #7647 for details).
-                 * The invalid state can be fixed by to re-focusing the editText.
-                 */
-                editText.clearFocus();
-                editText.requestFocus();
-
-                // Try again
-                imm.showSoftInput(editText, InputMethodManager.SHOW_FORCED);
-            }
```

Comment:
Are you sure we can remove this safely? #7647

Reply:
Thanks for the linked issue. The issue reoccurred on an API 23 emulator, so I reverted it. Only the system UI visibility flags have been replaced.